Formula/icoutils.rb

```diff
@@ -25,7 +25,18 @@ class Icoutils < Formula
 
   depends_on "libpng"
 
+  on_monterey :or_newer do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   def install
+    inreplace "common/Makefile.am", "libcommon_a_LIBADD", "libcommon_la_LIBADD"
```

Comment:
We should upstream this. I suggest checking with @fxcoudert for why this is the correct thing to do to help convince upstream.

Reply:
`_a_LIBADD` runs `ar`, which is stupid and will add the `.a` file to the other `.a` library. `_la_LIBADD` runs `libtool`, which knows how to unpack the static archive before adding the objects.